cmd/manager/main.go

```diff
@@ -313,13 +318,12 @@ func execute() {
 			os.Exit(1)
 		}
 	}
-
-	// TODO (sabo): re-enable when webhooks are usable
-	// log.Info("Setting up webhooks")
-	// if err := webhook.AddToManager(mgr, roles, newWebhookParameters); err != nil {
-	// 	log.Error(err, "unable to register webhooks to the manager")
-	// 	os.Exit(1)
-	// }
+	if viper.GetBool(EnableWebhooksFlag) {
+		if err = (&estype.Elasticsearch{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
This only validates Elasticsearch resources but we used to have validations on (license) Secrets as well. Just flicking through the docs I _think_ it should be possible to re-introduce them, but it might be worth revisiting the decision to validate secrets because it can potentially have a big impact on the stability of the k8s installation. Maybe worth re-introducing a license CRD?